### PR TITLE
Add coverage for create location functionality for CLI

### LINKED ISSUE #2420: Automate location create command in cli
Please automate:
$ hammer location create --help
Usage:
    hammer location create [OPTIONS]

Options:
 --compute-resource-ids COMPUTE_RESOURCE_IDS Comma separated list of values.
 --compute-resources COMPUTE_RESOURCE_NAMES  Comma separated list of values.
 --config-template-ids CONFIG_TEMPLATE_IDS   Comma separated list of values.
 --config-templates CONFIG_TEMPLATE_NAMES    Comma separated list of values.
 --description DESCRIPTION                    
 --domain-ids DOMAIN_IDS                     Numerical ID or domain name
                                             Comma separated list of values.
 --domains DOMAIN_NAMES                      Comma separated list of values.
 --environment-ids ENVIRONMENT_IDS           Comma separated list of values.
 --environments ENVIRONMENT_NAMES            Comma separated list of values.
 --hostgroup-ids HOSTGROUP_IDS               Comma separated list of values.
 --hostgroups HOSTGROUP_NAMES                Comma separated list of values.
 --media MEDIUM_NAMES                        Comma separated list of values.
 --media-ids MEDIA_IDS                       Media IDs
                                             Comma separated list of values.
 --medium-ids MEDIUM_IDS                     Comma separated list of values.
 --name NAME                                  
 --realm-ids REALM_IDS                       Numerical ID or realm name
                                             Comma separated list of values.
 --realms REALM_NAMES                        Comma separated list of values.
 --smart-proxies SMART_PROXY_NAMES           Comma separated list of values.
 --smart-proxy-ids SMART_PROXY_IDS           Comma separated list of values.
 --subnet-ids SUBNET_IDS                     Comma separated list of values.
 --subnets SUBNET_NAMES                      Comma separated list of values.
 --user-ids USER_IDS                         Comma separated list of values.
 --users USER_LOGINS                         Comma separated list of values.
 -h, --help                                  print help

### PR DESCRIPTION
**Pull request by oshtaier**
Closes #2420
```
nosetests tests/foreman/cli/test_location.py
...S......................S..........
----------------------------------------------------------------------
Ran 37 tests in 530.900s
```
Added two BZ and added possible coverage for them:
https://bugzilla.redhat.com/show_bug.cgi?id=1233612
https://bugzilla.redhat.com/show_bug.cgi?id=1234287

We can definitely improve general approach for testing provided in the code, but it seems for me as a best one from QA perspective and also we can achieve good coverage for most plausible scenarios without adding so many redundant cases

### CODE REVIEW

File: tests/foreman/cli/test_location.py [hunk 1]
```diff
@@ -0,0 +1,379 @@
...
+    make_user,
+)
+from robottelo.common.decorators import data, skip_if_bug_open
+from robottelo.test import CLITestCase
+
+
+@ddt
+class TestLocation(CLITestCase):
+    """Tests for Location via Hammer CLI"""
+    # TODO Add coverage for smart_proxy and realm once we can create ssh tunnel
```
Comment (Ichimonji10) [inline tests/foreman/cli/test_location.py]: Is there a GitHub bug for this?

**Comment by Ichimonji10** [comment]
ACK

**Comment by elyezer** [comment]
ACK

File: tests/foreman/cli/test_location.py (hunk 1 above)
Comment (oshtaier) [inline tests/foreman/cli/test_location.py]: #2487

Comment (Ichimonji10) [inline tests/foreman/cli/test_location.py]: :+1:

**Pull request merged**